lib/Screens/specific_story.dart

```diff
@@ -0,0 +1,201 @@
+                            throw 'Could not launch ${snapshot.data!.title}';
+                          }
+                        },
+                        child: Padding(
+                          padding: const EdgeInsets.all(20.0),
+                          child: Text(
+                            snapshot.data!.title,
+                            textAlign: TextAlign.center,
+                            style: const TextStyle(
+                              fontSize: 25,
+                              fontWeight: FontWeight.bold,
+                              decoration: TextDecoration.underline,
+                              color: Colors.blueAccent,
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                    if (snapshot.data!.type == 'story')
+                      SliverPadding(
+                        padding: const EdgeInsets.all(15.0),
+                        sliver: SliverToBoxAdapter(
+                          child: Html(
+                            data: snapshot.data!.text,
+                            style: {"body": Style(fontWeight: FontWeight.bold)},
+                            onLinkTap: _launchURL,
+                          ),
+                        ),
+                      ),
+                    SliverPadding(
+                      padding: const EdgeInsets.only(right: 20.0),
+                      sliver: SliverToBoxAdapter(
+                        child: GestureDetector(
+                          onTap: () {
+                            Navigator.push(
+                                context,
+                                MaterialPageRoute(
+                                  builder: (context) => ShowUser(
+                                    user: snapshot.data!.by,
+                                  ),
+                                ));
+                          },
+                          child: Text(
+                            snapshot.data!.by,
+                            textAlign: TextAlign.right,
+                            style: const TextStyle(
+                              decoration: TextDecoration.underline,
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                    SliverList(
+                      delegate: SliverChildBuilderDelegate(((context, index) {
+                        if (index < commentIDs.length) {
+                          Future<Comment> comment =
+                              fetchComments(commentIDs[index]);
+                          return FutureBuilder<Comment>(
+                            future: comment,
+                            builder: ((context, snapshot) {
+                              if (snapshot.hasData) {
+                                //print(snapshot.data!.kids);
+                                return Padding(
```

Comment:
This is another good case for https://github.com/DavBot02/World-Clock/pull/5/files#r920583835 to break up the indentation.